src/defaults.js

```diff
@@ -138,6 +138,36 @@ var bookingConfirmDecline = {
 
 };
 
+// Preset: bookingGraph = 'group_customer'
+var bookingGroupCustomer = {
+
+  timekitCreateBooking: {
+    graph: 'group_customer',
+    action: 'create',
+  },
+  localization: {
+    strings: {
+      successMessageBody: "You have reserved a seat at the booking. Check your email for further instructions. <br /><br />Have a great day!"
```

Comment:
"You have reserved a seat at the booking." This sounds a little funny. why not keep it simple with something like:
"Your seat has been reserved, please check your email for instructions."